cmd/training-operator.v1/main.go

```diff
@@ -74,6 +75,7 @@ func main() {
 		"If set, it only monitors kubeflow jobs in the given namespace.")
 	flag.IntVar(&monitoringPort, "monitoring-port", 9443, "Endpoint port for displaying monitoring metrics. "+
 		"It can be set to \"0\" to disable the metrics serving.")
+	flag.IntVar(&controllerThreads, "controller-threads", 10, "Number of worker threads used by the controller.")
```

Comment:
Should we set the `controllerThreads` per each framework since I don't think users create many Jobs using all frameworks simultaneously?

This means we introduce `--tfjob-threads`, `pytorchjob-threads`, and so on here instead of the `--controller-threads`.

@johnugeorge wdyt?

Reply:
Users now can enable and disable schemes that supported by the operator instance.

I think we can keep the command line options simple at first and update it to a more sophisticated version when real cases pump in.

Reply:
@zw0610 I see; we can select schemes in the following. Thanks for clarifying!

https://github.com/kubeflow/training-operator/blob/69813fbc37f33857e59044d14caa7e2765851d98/cmd/training-operator.v1/main.go#L69-L70